week1/Q1.py

```diff
@@ -0,0 +1,21 @@
+def normalize_word(word):
+	word = word.lower()
+	word = sorted(word)
+	return word
+
+
+def if_permutation(w1, w2):
```

Comment:
Since this is your main function, it would be nice to add a comment describing what it is mean to do, what its argument are meant to be, and what the return value represents.